devices/src/virtio/block.rs

```diff
@@ -707,13 +706,23 @@ mod tests {
 
             let epoll_config = EpollConfig::new(0, epoll_raw_fd, sender);
 
-            let f: File = tempfile().unwrap();
-            f.set_len(0x1000).unwrap();
+            let tmp_f = TempFile::new("/tmp/").unwrap();
+            tmp_f.as_file().set_len(0x1000).unwrap();
 
             // Rate limiting is enabled but with a high operation rate (10 million ops/s).
             let rate_limiter = RateLimiter::new(0, None, 0, 100_000, None, 10).unwrap();
             DummyBlock {
-                block: Block::new(f, is_disk_read_only, epoll_config, Some(rate_limiter)).unwrap(),
+                block: Block::new(
+                    OpenOptions::new()
+                        .read(true)
+                        .write(true)
```

Comment:
Shouldn't we add open options based on `is_disk_read_only` flag?

Reply:
Yes, I agree. I've updated, please take a look: [devices/src/virtio/block.rs:716](https://github.com/firecracker-microvm/firecracker/blob/145621d1bf024f5a692288b22e762de8ea0b9534/devices/src/virtio/block.rs#L716).